packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -454,6 +418,46 @@ contract Deploy is Deployer {
         }
     }
 
+    /// @notice Deploy all of the OP Chain specific contracts
```

Comment:
big diff, but all I did was move this function down to make the ordering more intuitive.